### DISCUSSION TITLE
Memory leakage

### DISCUSSION THREAD

**Discussion by leonballonigomes** [bloomberg/memray#256] [Q&A] [answered]
Good day, everyone.

Please, could you help me understand if it`s possible to evaluate my application memory leakage using memray?

Thank you.

**Comment by godlygeek** [comment, marked as answer]
You certainly can. You'd run your program with something like:
```
memray run -f -o memray_capture.bin --trace-python-allocators yourfile.py
```
and then you can generate a report from that capture file using:
```
memray flamegraph --leaks memray_capture.bin
```
and then open that report in a web browser. See [the `memray flamegraph`](https://bloomberg.github.io/memray/flamegraph.html) docs and our [docs on Python allocators](https://bloomberg.github.io/memray/python_allocators.html).

**Comment by leonballonigomes** [comment]
Thank you!
